UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -93,6 +93,60 @@ class BTShopperInsightsClient_Tests: XCTestCase {
         }
     }
     
+    func testGetRecommendedPaymentMethods_whenEligibleInPayPalNetworkTrue_returnsOnlyPayPalRecommended() async {
+        do {
+            let mockPayPalRecommendedResponse = BTJSON(
+                value: [
+                    "eligible_methods": [
+                        "paypal": [
+                            "can_be_vaulted": true,
+                            "eligible_in_paypal_network": true,
+                            "recommended": true,
+                            "recommended_priority": 1
+                        ]
+                    ]
+                ]
+            )
+            mockAPIClient.cannedResponseBody = mockPayPalRecommendedResponse
+            let result = try await sut.getRecommendedPaymentMethods(request: request)
+            XCTAssertTrue(result.isPayPalRecommended)
+            XCTAssertFalse(result.isVenmoRecommended)
+            XCTAssertTrue(result.isEligibleInPayPalNetwork)
+            XCTAssertEqual(mockAPIClient.postedAnalyticsEvents.last, "shopper-insights:get-recommended-payments:succeeded")
+        } catch let error as NSError {
+            XCTFail("An error was not expected.")
+        }
+    }
+    
```

Comment:
Do we want to add if Venmo's eligibleInPayPalNetWork is true?

Reply:
Other than that, it looks great!

Reply:
for sure, addressed in 42e7947210a0be8ea597e3c467c4df0ee7349978